tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -308,15 +335,22 @@ def _test_cpu_rdmsr(context):
     # Guest Kernel 5.10 sets up some MSRs differently.
     if context.kernel.name() == "vmlinux-5.10.bin":
         guest_msrs_5_10 = {
-            # See https://github.com/torvalds/linux/commit/1db2a6e1e29ff994443a9eef7cf3d26104c777a7
-            "0x3a": "0x1",  # MSR_IA32_FEAT_CTL
             # See https://github.com/torvalds/linux/commit/229b969b3d38bc28bcd55841ee7ca9a9afb922f3
             "0x808": "0x10",  # IA32_X2APIC_TPR
             "0x80a": "0x10",  # IA32_X2APIC_PPR
             # `arch/x86/include/asm/irq_vectors.h` to see how LOCAL_TIMER_VECTOR changed
             "0x832": "0x400ec",  # IA32_X2APIC_LVT_TIMER
         }
-
+        guest_msrs_5_10_intel = {
+            # See https://github.com/torvalds/linux/commit/1db2a6e1e29ff994443a9eef7cf3d26104c777a7
+            # Referring to above link, MSR_IA32_FEAT_CTL (0x3A) is set to 1 in init_ia32_feat_ctl()
+            # which is called from init_intel().
+            # The same is not done in amd_init() and hence we do this only for Intel.
```

Comment:
Do we still need it, provided that the baselines are now separate for Intel and AMD?